phalcon/assets/collection.zep

```diff
@@ -127,7 +129,7 @@ class Collection implements \Countable, \Iterator
 	 * @param array attributes
 	 * @return \Phalcon\Assets\Collection
 	 */
-	public function addJs(string! path, var local = null, boolean filter = true, attributes = null) -> <Collection>
+	public function addJs(string! path, var local = null, boolean filter = true, attributes = null, version = null) -> <Collection>
```

Comment:
Any idea why version here is always null even that is called like:

`$assets->addJs(PATH_DATA.'assets/script1.js', true, false, null, "asdsadasdsa");`

???